sdk/src/main/java/co/omise/android/ui/AuthorizingPaymentActivity.java

```diff
@@ -103,12 +112,24 @@ public boolean shouldOverrideUrlLoading(WebView view, String url) {
                     setResult(RESULT_OK, resultIntent);
                     finish();
                     return true;
+                } else if (verifier.verifyExternalURL(uri)) {
+                    Intent externalIntent = new Intent(Intent.ACTION_VIEW, uri);
+                    startActivityForResult(externalIntent, REQUEST_EXTERNAL_CODE);
+                    return true;
                 } else {
                     return false;
                 }
             }
         });
     }
-}
 
+    @Override
+    protected void onActivityResult(int requestCode, int resultCode, Intent data) {
+        super.onActivityResult(requestCode, resultCode, data);
+        if (requestCode == REQUEST_EXTERNAL_CODE && resultCode == RESULT_OK) {
```

Comment:
Should we verify the result in the returned Intent too?

Reply:
The result is depend on external app. By default can check only `RESULT_OK` or `RESULT_CANCELED`. So, merchant developer must handle `Intent` result back by themselves na krub.